src/test/java/com/bugsnag/BugsnagTest.java

```diff
@@ -339,6 +342,7 @@ public void beforeNotify(Report report) {
         assertFalse(bugsnag.notify(new Throwable()));
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Probably worth a comment that it's the deprecated method that's under test. We should also add a test `testEndpoints` that tests `setEndpoints`.

Reply:
`ConfigurationTest` covers `setEndpoints`, I'll update with an inline comment